assets/dev/js/frontend/elements-handler.js

```diff
@@ -1,45 +1,29 @@
-import Accordion from './handlers/accordion';
-import Alert from './handlers/alert';
-import Counter from './handlers/counter';
-import Progress from './handlers/progress';
-import Tabs from './handlers/tabs';
-import Toggle from './handlers/toggle';
-import Video from './handlers/video';
-import ImageCarousel from './handlers/image-carousel';
-import TextEditor from './handlers/text-editor';
+import globalHandler from './handlers/global';
```

Comment:
Consider rename to `elements-async-handlers`. and `external-handlers.js` => `elements-sync-handlers`. 
or any names that explains the different between `elements-handler.js` vs `external-handlers.js`

Reply:
Fixed.